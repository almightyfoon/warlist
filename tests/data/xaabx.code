c2101b_-0dlTlVlUe4lVlM1G1L3t3B2_m2___-0Om11U1U1rme3k3kjCjC3E3r3r7z2_3G3H3G3H
