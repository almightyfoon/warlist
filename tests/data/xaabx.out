https://conflictchamber.com/#c2101b_-0dlTlVlUe4lVlM1G1L3t3B2_m2___-0Om11U1U1rme3k3kjCjC3E3r3r7z2_3G3H3G3H

Protectorate Army - 75 / 75 points
[Theme] The Creator's Might

[Cyrenia 1] High Exemplar Cyrenia [+28]
 - Judicator [34]
Exemplar Warder [0(5)]
Initiate Tristan Durant [0(4)]
 - Judicator [34]
Scrutator Potentate Severius [0(6)]
 - Redeemer [11]
 - Vigilant [9]
Vassal Mechanik [1]
Wrack [1]
Choir of Menoth (min) [4]
Visgoth Juviah Rhoven & Honor Guard [9]

Protectorate Army - 75 / 75 points
[Theme] The Faithful Masses

[Harbinger 1] The Harbinger of Menoth [+27]
 - Crusader [10]
 - Crusader [10]
 - Devout [9]
 - Hierophant [3]
Allegiant of the Order of the Fist [3]
Allegiant of the Order of the Fist [3]
Champion of the Order of the Wall [8]
Champion of the Order of the Wall [8]
High Paladin Dartan Vilmon [6]
Paladin of the Order of the Wall [0(4)]
Paladin of the Order of the Wall [4]
Rhupert Carvolo, Piper of Ord [4]
Choir of Menoth (min) [4]
Idrian Skirmishers (max) [15]
 - Idrian Skirmisher Chieftain & Guide [0(5)]
Idrian Skirmishers (max) [15]
 - Idrian Skirmisher Chieftain & Guide [0(5)]
